static_report/src/components/Serve/ServeIndex.tsx

```diff
@@ -0,0 +1,234 @@
+import {
+  Flex,
+  Heading,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  Tab,
+  Tabs,
+  TabList,
+  TabPanels,
+  TabPanel,
+  LinkBox,
+  LinkOverlay,
+  Checkbox,
+  Box,
+  Text,
+  Button,
+  Center,
+} from '@chakra-ui/react';
+
+import { useState } from 'react';
+import { Main } from '../shared/Main';
+import { formatReportTime } from '../../utils/formatters';
+
+function useForceUpdate() {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [value, setValue] = useState(0);
+  return () => setValue((value) => value + 1);
+}
+
+const style = {
+  opacity: 0.96,
+  backgroundColor: '#F8F8F8',
+  border: '1px solid #E7E7E7',
+  borderRadius: '15px',
+  padding: '15px',
+  position: 'fixed',
+  bottom: '10px',
+  height: '80px',
+  width: '35%',
+  zIndex: 3,
+};
+
+function StickyFooter({ checkState, onCompareClick }) {
+  return (
+    <Center>
+      <Box sx={style}>
+        <Center>
+          <Box w={70} textAlign={['left']}>
+            <Text>Base: </Text>
+            <Text>Target: </Text>
+          </Box>
+          <Box mr={30}>
+            <Text>{checkState[0]}</Text>
+            <Text>{checkState[1]}</Text>
+          </Box>
+          <Box m={'auto 0'}>
+            <Button onClick={onCompareClick}>Compare</Button>
+          </Box>
+        </Center>
+      </Box>
+    </Center>
+  );
+}
+
+export function ServeIndex({ data }) {
```

Comment:
Can we call this `IndexReport`, to keep consistent with `SingleReport` and `ComparisonReport`? (`<NAME>Report`)

Reply:
Yes. changed